test/Npgsql.Tests/TestUtil.cs

```diff
@@ -362,6 +362,15 @@ public static bool IsSequential(this CommandBehavior behavior)
             => (behavior & CommandBehavior.SequentialAccess) != 0;
     }
 
+    public static class NpgsqlCommandExtensions
+    {
+        public static void EnsureInProgress(this NpgsqlCommand command)
```

Comment:
Small naming nit, because Ensure* usually means "throw if not"

```suggestion
        public static void WaitUntilCommandIsInProgress(this NpgsqlCommand command)
```